zio-schema-examples/shared/src/main/scala/dev/zio/schema/example/example3/Example3.scala

```diff
@@ -46,7 +46,7 @@ private[example3] object Domain {
 
 }
 
-object Example3 extends zio.App {
+object Example3 extends zio.ZIOAppDefault {
```

Comment:
Might read better if you just imported `ZIOAppDefault` so it could be `Example3 extends ZIOAppDefault`. One nice thing about the name change here is it no longer conflicts with `scala.App`.